### PR TITLE
stm32mp1: add dev signing keys

### PR DESCRIPTION
**Pull request by mike-scott**
Refer to https://docs.foundries.io/latest/reference-manual/security/secure-boot-stm32mp1.html
for how these keys were generated and commands to use them.

Signed-off-by: Michael Scott <mike@foundries.io>

### CODE REVIEW

**Review by Tim-Anderson — APPROVED** [review]
LGTM

File: security/stm32mp1/privateKey.pem [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```
Comment (ricardosalveti) [inline security/stm32mp1/privateKey.pem:1]: This file is encrypted, what is the default password for it?

Comment (mike-scott) [reply security/stm32mp1/privateKey.pem:1]: Woops: `fio`

**Review by igoropaniuk — APPROVED** [review]

**Review by MrCry0 — APPROVED** [review]
LGTM

**Pull request merged**